test/json/any_json_helper.cpp

```diff
@@ -70,7 +70,7 @@ struct any_json_helper_t {
     static const T& as(const value_type& x) { return any_cast<const T&>(x); }
         
     static void move_append(object_type& obj, key_type& key, value_type& value) {
-        obj.emplace(move(key), move(value));
+        obj.emplace(std::move(key), std::move(value));
```

Comment:
The change in the header is correct - where is the ambiguity coming from in the .cpp file?